### PR TITLE
Add xfab

### PR DESCRIPTION
**Pull request by AxelHenningsson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AxelHenningsson** [comment]
I confirm that I will maintain this.

**Comment by AxelHenningsson** [comment]
@conda-forge/staged-recipes : Think this is ready to be merged 😄 🚀 !

File: recipes/xfab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xfab-{{ version }}.tar.gz
+  sha256: 89c14f5feae2259fbed23d26264d0ac90076658b15c2f348cbb489fb208d6950
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
[the remark this answers is not in the document]
Comment (AxelHenningsson) [reply recipes/xfab/meta.yaml:13]: @[ocefpaf](https://github.com/ocefpaf) Thanks for the input. Think we are ready to fly now 🚀

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/xfab/meta.yaml:13]: Ah. Grayskull did the right thing and my suggestion was not correct. This package has scripts, see https://github.com/FABLE-3DXRD/xfab/blob/master/setup.py#L18, and those do not work well with noarch. When you can, please remove the noarch in the feedstock, rerender, and bump the build number.

Sorry, scripts are a bit rare these days but still a headache that I should have paid attention.

**Pull request merged**